app/config/config_dev_event_replay.yml

```diff
@@ -3,6 +3,3 @@ imports:
 
 swiftmailer:
     disable_delivery: true
-
-surfnet_stepup_middleware_command_handling:
-    warn_on_missing_email_template: false
```

Comment:
These settings have already been removed in the past (2a7e965219188b29d14cabfea0fcabaca3d333af), but apparently they were not removed from the event_replay specific configuration files.